po/de.po

```diff
@@ -11702,6 +11729,7 @@ msgstr "Gewählte Einheit(en) vernichten"
 msgid "Toggle god Mode Status"
 msgstr "God-Mode-Status umschalten"
 
+# Sollte das nicht eher "Zeige Einstellungsbildschirm" heißen? -alfred007
 #: src/keymap.cpp:456
 msgid "Display Options Screen"
 msgstr "Zeige Darstellungsoptionen"
```

Comment:
Gute Frage ^.^ Wie/wo wird das im Spiel genutzt?

Reply:
Das ist im Debug-Menü. Mit der Tastenkombination LCTRL und "O" öffnest Du den Debug-Bildschirm mit der Möglichkeit, den Level zu verändern. Ich habe den Begriff bei der Erstellung der Datei einfach sinngemäß übersetzt. Nach Deiner Frage habe ich nachgesehen wo das genutzt wird. Und somit ist "Zeige Einstellungsbildschirm" tatsächlich die korrekte Übersetzung.

Reply:
Ich kann im Debugbildschirm keine solche Möglichkeit erkennen. Ist das in 3.2.3 noch nicht drin?

Reply:
Ich weiß auch nicht genau, wo die Übersetzung im Spiel auftaucht. Ich habe in src/keymap.cpp:456 nachgesehen und die dort angegebene Tastenkombination im Spiel ausprobiert. Anschließend hat sich ein neues Fenster geöffnet mit diversen Tabs mit denen Du am aktuellen Level rumspielen kannst. "Zeige Einstellungsbildschirm" ist daher die passendere Übersetzung als "Zeige Darstellungsoptionen".

Reply:
Wenn du davon einen Screenshot hast (Englisch), würde mich das freuen. Ich vertraue dir da, will aber einfach sehen, wie das genau aussieht ^.^ Du solltest hier in den Kommentaren auch Bilder reinziehen können.

Reply:
Was ich vergessen habe ist, dass sich der Bildschirm außerhalb des Spiels in einem neuen Tab öffnet. Daher der schwarze Hintergrund.

![Screenshot 2019-04-07 18 01 56](https://user-images.githubusercontent.com/47194838/55686293-a6665400-595f-11e9-9089-c69133609a5e.png)

Reply:
Interessant, scheinen die kräftig aufpoliert zu haben. Bei welchem Tab steht dann "Display Options Screen"?

Reply:
Das weiß ich nicht. Ich habe die Stelle im Spiel noch nicht gefunden wo das steht. Da müsstest Du mal pastdue oder Forgon fragen.
 Es könnte sogar sein, dass die Übersetzung überflüssig ist, da auch das englische Original nirgends im Spiel auftaucht. Das sollte uns aber nicht daran hindern, den Text korrekt zu übersetzen. Vielleicht ist es auch nur geplant, dass noch einzuarbeiten und man hat vorsorglich schon mal die Übersetzung eingebaut.

Reply:
Alles klar. :)